.travis.yml

```diff
@@ -15,5 +15,5 @@ before_script:
 
 script:
     - ./vendor/bin/php-cs-fixer fix --dry-run
-    - ./vendor/bin/phpstan analyse --level=7 lib
```

Comment:
The path is already configured, see https://github.com/dantleech/fink/blob/1ac3c48df3b1bbcdab103313ec44ec8f5fa7f3b4/phpstan.neon#L2